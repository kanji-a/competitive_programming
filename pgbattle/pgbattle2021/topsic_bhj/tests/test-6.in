3 199
2 1 2
