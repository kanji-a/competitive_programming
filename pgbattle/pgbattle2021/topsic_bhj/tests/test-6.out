99 9 91
